### ISSUE TITLE
Can RaySession handle several jack server (with different name) ?

### ISSUE THREAD

**Issue by newlaurent62** [Houston4444/RaySession#109]
Hello,

Can RaySession handle several jack server name ?

I can start two jack server instances with the following command lines (I deactivate Service D-Bus in qjackctl and created two presets U22 and XR18):

`qjackctl -n U22 & qjackctl -n XR18 &`

But JackPatch in RaySession stay inactive (It seems to wait for DEFAULT Jack server ?)

BR,
Laurent

**Comment by newlaurent62** [comment]
Well i just remembered that i don't use the last version ... I'll download and try the last official version.

**Comment by newlaurent62** [comment]
I answer my question: By using JACK_DEFAULT_SERVER env variable we can set the server name to use.

`export JACK_DEFAULT_SERVER=XR18;raysession`

It seems to work fine with ray-jackpatch.

BR,
Laurent

**Comment by Houston4444** [comment]
Hi. That is exactly what I was about to ask.
I didn't know there was the possibility to run simultaneously two JACK servers. I was asking myself how a program knows on which server should it be running, I have the answer, thanks.

Do you know if there is the possibility to run a dummy server where hardware in/outs are connected to a client in a true server ?
This would enable the possibility to create rooms.

**Comment by newlaurent62** [comment]
Yes it is possible to connect several jack server (dummy or not) using Jamulus software (and may be sonobus , jacktrip ...).

https://github.com/jamulussoftware/jamulus/discussions/1397

**Comment by newlaurent62** [comment]
Jamulus can connect to windows, Mac and Linux and can be started several times on a single computer (Linux only) with several jack servers.

**Comment by newlaurent62** [comment]
but unfortunately the audio is compressed in Jamulus ...

**Comment by newlaurent62** [comment]
Hello,

Jacktrip could do the job with uncompressed signal. It is a command line but you can use qjacktrip to learn about jacktrip options. 

Jacktrip works in hub mode and in peer to peer. It is very flexible and multiplateform !

**Comment by Houston4444** [comment]
I just added to possibility in patchbay to get one different patchbay per daemon. Could you confirm it works with several jack instances ?

**Comment by newlaurent62** [comment]
Hello Houston4444,

I started in a terminal: 

```
export JACK_DEFAULT_SERVER=DUMMY
qjackctl -n DUMMY & raysession &
Jamulus --clientname "DUMMY" &
```

In a second one :
```
export JACK_DEFAULT_SERVER=default
qjackctl & raysession &
Jamulus --clientname "default" &
```
And it works. I have two jack server patchbays with expected clients and ports. 

Thanks !
